cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
```

Comment:
Ah my bad. I totally forgot we were on python 3.5 and unit test.mock is available. Will fix this.